src/shmem_team.c

```diff
@@ -310,15 +320,22 @@ int shmem_internal_team_split_strided(shmem_internal_team_t *parent_team, int PE
 
     shmem_internal_team_release_psyncs(parent_team, SYNC);
 
+    /* This OR reduction assures all PEs return the same value.  */
+    psync = shmem_internal_team_choose_psync(parent_team, REDUCE);
+
+    shmem_internal_op_to_all(team_ret_val_reduced, team_ret_val, 1, sizeof(int),
+                             parent_team->start, parent_team->stride, parent_team->size, NULL,
+                             psync, SHM_INTERNAL_BOR, SHM_INTERNAL_INT);
```

Comment:
Would `SHMEM_INTERNAL_MAX` be a better reduction op? That is guaranteed to lead to a valid error code that is meaningful, whereas OR'ing multiple error codes could lead to something totally different.

Reply:
Yeah, good point.